compiler/bytecode/instruction.go

```diff
@@ -116,11 +115,13 @@ func (i *Instruction) AnchorLine() int {
 }
 
 // Line returns instruction's line number
+// TODO: needs to change the func to simple public variable
```

Comment:
I don't think we need to add these TODO comments

Reply:
Removed